### PR TITLE
Feature/e UI 3737 new pba unhappy path

### PR DESCRIPTION
**Pull request by tarikdogan-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-3737

### Change description ###

New PBA's-Select radio button and click on Continue (Multiple PBA's)- Unhappy path

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: src/org-manager/components/new-pbas-info/new-pbas-info.component.spec.ts [hunk 1]
```diff
@@ -25,19 +51,43 @@ describe('NewPBAsInfoComponent', () => {
...
     component.onSubmitForm();
     expect(component.submitForm.emit).toHaveBeenCalled();
   });
 
+  it('should not submitForm if there is no radiobutton selected', () => {
+    spyOn(component.submitForm, 'emit').and.callThrough();
+    const submitButton = fixture.debugElement.nativeElement.querySelector('button[type="submit"]');
+    submitButton.click();
+    fixture.detectChanges();
+    component.onSubmitForm();
```
Comment (paulhowes-HMCTS) [inline src/org-manager/components/new-pbas-info/new-pbas-info.component.spec.ts:87]: Same as my previous comment - Is component.onSubmitForm needed now as you are calling the submitButton click event on line 85.

File: src/org-manager/components/new-pbas-info/new-pbas-info.component.spec.ts [hunk 2]
```diff
@@ -25,19 +51,43 @@ describe('NewPBAsInfoComponent', () => {
...
   });
 
   it('should call the submitForm output event emitter when onSubmit is called', () => {
     spyOn(component.submitForm, 'emit').and.callThrough();
+    const radioButton = fixture.debugElement.nativeElement.querySelector('#PBA0101012_approve');
+    radioButton.click();
+    fixture.detectChanges();
+    const submitButton = fixture.debugElement.nativeElement.querySelector('button[type="submit"]');
+    submitButton.click();
     component.onSubmitForm();
```
[the remark this answers is not in the document]
Comment (tarikdogan-hmcts) [reply src/org-manager/components/new-pbas-info/new-pbas-info.component.spec.ts]: done

File: src/org-manager/components/new-pbas-info/new-pbas-info.component.spec.ts (hunk 1 above)
Comment (tarikdogan-hmcts) [reply src/org-manager/components/new-pbas-info/new-pbas-info.component.spec.ts]: done

**Review by paulhowes-HMCTS — APPROVED** [review]
Approved after changes.

**Pull request merged**